### PR TITLE
fixed last solution in Basic Bonfires

### PR DESCRIPTION
**Pull request by krantzinator**
address issue #4112

### CODE REVIEW

**Comment by ltegman** [comment]
Thanks for the PR! Unfortunately it looks like this solution still won't pass. There are tests that pass in an unordered array that needs to be sorted first.

**Comment by krantzinator** [comment]
I pulled this from @dting's solutions. I personally haven't gotten to this
bonfire yet so am not sure of the correct solution. I'll look at it now
though and see if I can come up with the solution.

**Comment by krantzinator** [comment]
This one passes the tests. Lesson learned: don't blindly copy-paste :stuck_out_tongue_closed_eyes:

**Comment by ltegman** [comment]
Much better :smile: Not to be too picky, but would you mind squashing both your commits into one using `get rebase`? The wiki has an [article](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase) on it if you've not done it before.

File: seed/challenges/basic-bonfires.json [hunk 1]
```diff
@@ -664,7 +664,7 @@
         "Array.sort()"
       ],
       "solutions": [
-        "function where(arr, num) {\n  // Find my place in this sorted array.\n  return num;\n}\n\nwhere([40, 60], 50);\n"
+        "function where(arr, num) {\n  arr = arr.sort(function(a, b){return a-b});\n  for (i = 0; i < arr.length; i++) {\n    if (arr[i] >= num) {\n       return i;\n    }\n  }\n  return arr.length;\n}\n\nwhere([40, 60], 50);\n"
       ],
```
Comment (BerkeleyTrue) [inline seed/challenges/basic-bonfires.json]: Only include the solution, not the call to the function.

**Comment by abhisekp** [comment]
@krantzinator the PR has conflicts.

**Comment by krantzinator** [comment]
man this really turned into a party!
@BerkeleyTrue the original instructions say to include the function, see here in issue #3836. Does this need updated?
@abhisekp do you mean the conflict @ltegman pointed out, or a new one? Could you point it/them out so I can fix them?
@ltegman thanks for the pointer! I'll look that up and get it sorted. I have not done it before - thanks for being patient with me.

**Comment by BerkeleyTrue** [comment]
@krantzinator do not include the function call.

**Comment by abhisekp** [comment]
@krantzinator fetch the staging branch, rebase into your branch and then force push after resolving the conflicts.

**Comment by krantzinator** [comment]
@BerkeleyTrue 10-4. Do you want me to create new issues to remove the function calls from all the [basic] (https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3844), [intermediate] (https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3869/files), and [advanced] (https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3841/files) solutions? Or does it make more sense to expand this issue and include them here?

**Comment by BerkeleyTrue** [comment]
@krantzinator they should be in separate PR's

**Comment by krantzinator** [comment]
this has been blocked; closing PR

**Comment by ltegman** [comment]
Blocked just means that the PR has issues to resolve before it can be merged. If you're still interested in this issue feel free to reopen the PR!

**Comment by SaintPeter** [comment]
@krantzinator Do you want to fix this up?  We still need the solution.

**Comment by krantzinator** [comment]
@SaintPeter sure. how do I tell why it's failing the Travis CI build?

**Comment by krantzinator** [comment]
oh geez nevermind. this whole thing has been a communication cluster, but i see what's actually needed now.

**Pull request closed without merging**